pubcrawler/article.py

```diff
@@ -27,7 +27,7 @@ def get_text_from_tags(self, tag_name):
         if tags is not None:
             tags_text = []
             for tag in tags:
-                tags_text.append(tag.get_text("\n\n"))
```

Comment:
Sounds good. I think I used a double line break so that paragraphs were visually separated.